pgtopoeditor.py

```diff
@@ -335,9 +335,9 @@ def doDropOrphanedTopoGeoms(self):
             (toponame, layer_id) = cur.fetchone()
 
             # delete orphaned geoms...
-            cur.execute('DELETE FROM "' + toponame
-              + '"."relation" WHERE layer_id = %s AND topogeo_id NOT IN ( SELECT id("'
-              + col + '") FROM "' + schema + '"."' + table + '")', (str(layer_id)))
+            cur.execute('DELETE FROM "' + toponame +
+              '"."relation" WHERE layer_id = %s AND topogeo_id NOT IN ( SELECT id("'
+              + col + '") FROM "' + schema + '"."' + table + '")', (str(layer_id),))
             QMessageBox.information(None, toolname, str(cur.rowcount)
```

Comment:
this sems to be formatting only change isn't it ?

Reply:
There's an extra comma at the end changing it from a str to a single item tuple. It was unable to parse the argument otherwise.

Reply:
who was unable to parse the argument ? I'm confused

Reply:
Passing just `(str(layer_id))` to `cur.execute()` will pass the str as an array of char, I think. But adding a comma as so `(str(layer_id),)` passes a single item tuple instead.